### DISCUSSION TITLE
All bookings from a certain origin (VISA debit card) are assigned to the same expense acount and category

### DISCUSSION THREAD

**Discussion by meinnamehier** [firefly-iii/firefly-iii#8049] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am running Firefly on an Unraid box, and since the last update (to 6.13.0), all the transactions that were done with my VISA debit card are linked to one expense account and have received the same category.
This was previously not the case and it is also not a problem with the import (I would say), since even data that was imported much earlier is affected.
I don't know how I can describe this any better, if need be I can attach a screenshot, but I don#t know which page would be most helpful.

### Debug information

Debug information generated at 2023-10-09 10:29:04 Europe/Berlin for Firefly III version **6.0.13**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.13 |
| Firefly III API | 2.0.4 |
| Build | 506, 11-06-2023 16:24:57 UTC |
| Base Build | 6, 10-06-2023 16:13:33 CEST |
| DB version | 19 (exp. 19) |
| PHP | `8.2.7` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Berlin |
| Browser TZ | Europe/Berlin |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 67108864 (64 MB) |
| Interface | apache2handler |
| Default language | de_DE |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2023-10-01 00:00:00 |
| Session end | 2023-10-31 23:59:59 |
| Session first | 2022-06-16 00:00:00 |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/115.0 |

### Expected behaviour

Transactions done with the Debit Card are linked to the correct expense account.

### Steps to reproduce

1. Go to the "Expense Bookins" page. 
2. See that all transactions from the Debit card show the same target account and category

**Comment by JC5** [comment]
Could you update to the latest version and try again?

**Comment by meinnamehier** [comment]
I am on 6.0.26 now and the issue is still there

**Comment by JC5** [comment]
> since even data that was imported much earlier is affected.

What do you mean by this? Transactions in the past are edited?

**Comment by meinnamehier** [comment]
I cannot say if they are edited, but I attached a screenshot with bookings from July 2022. This data was assigned correctly before, but now they are all on the same expense account
![grafik](https://github.com/firefly-iii/firefly-iii/assets/17614384/0a806ec8-80e1-4129-bee9-937418ebec5e)
And only few of those bookings were actually done to that expense account

**Comment by JC5** [comment]
The data importer doesn't change past transactions. I don't know what changed that. Perhaps a rule?

Either way, I'll need some info on the original import. Like logs from the importer.

**Comment by meinnamehier** [comment]
I am fairly certain it is not a rule, since I have never set up a rule like that. The only thing I have rules for is to change the category, and none of the rules would apply to these transactions. 
How can I access old logs from the importer? I looked into storage/logs on that container, but it was empty.

**Comment by JC5** [comment]
The importer doesn't keep old logs, it logs to stdout (`docker logs -f [container]`). What you could do with a new import is track the logs and send them to me. I'm curious what data your bank provides.

You haven't told me yet how you import. Nordigen, Salt Edge, CSV?

**Comment by meinnamehier** [comment]
I am using Nordigen, sent the log output per mail.

**Comment by mrwsl** [comment]
I'm experiencing a similar behaviour where I have problems mapping the debit amount correctly. Imports are based on CSV. Should I send logs as well?
